Source/GameBaseFramework/Public/Engine/SubSystems/GBFGameInstanceGameStateSystem.h

```diff
@@ -35,11 +35,10 @@ class GAMEBASEFRAMEWORK_API UGBFGameInstanceGameStateSystem : public UGameInstan
     void Initialize( FSubsystemCollectionBase & collection ) override;
     void UpdateCurrentGameStateFromCurrentWorld();
     bool IsStateWelcomeScreenState( const UGBFGameState * state ) const;
+    UGBFGameState * GetGameStateFromName( FName state_name ) const;

```

Comment:
why removal of const?

Reply:
because in some of the calls we need a non const game state